### PR TITLE
build: Use automatic signing for local development

### PR DESCRIPTION
**Pull request by adriansberg**
These changes increase the DX for local development. Setting the signing to be automatic and setting the app group based on the value in the .env file should make it so that the developer does not have to do any manual clicks in XCode, given that the development team is set up as it should be.

### CODE REVIEW

**Comment by mikaelbr** [comment]
I wonder if this should actually be MANUAL with correctly injected variables instead of doing it automatic. When being automatic we need all users to be added to Apple, but I don't think that is practical for all 🤔

File: ios/AtbAppIntent/AtbAppIntentDebug.entitlements [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <dict>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.no.mittatb</string>
+		<string>$(APP_GROUP_NAME)</string>
```
Comment (mikaelbr) [inline ios/AtbAppIntent/AtbAppIntentDebug.entitlements:7]: So these changes are probably good

**Comment by adriansberg** [comment]
Right, that makes sense. Looks like a provisioning profile has to be provided when it's set to "Manual". That might be tricky to inject?

**Comment by adriansberg** [comment]
Discarding this as it's not as straight forward as I thought.

**Comment by gorandalum** [comment]
I think on issues like this it would be beneficial to discuss with @jorelosorio, as he has quite a good understanding of the ios setup.

**Pull request closed without merging**